packages/jest-config/src/resolveConfigPath.ts

```diff
@@ -95,3 +119,15 @@ const makeResolutionErrorMessage = (
   `traverse directory tree up, until it finds one of those files in exact order: ${JEST_CONFIG_EXT_ORDER.map(
     ext => `"${getConfigFilename(ext)}"`,
   ).join(' or ')}.`;
+
+const makeMultipleConfigsWarning = (configPaths: Array<Config.Path>) =>
+  [
+    '● Multiple configurations found:',
+    ...configPaths.map(configPath => `    * ${configPath}`),
```

Comment:
```suggestion
    ...configPaths.map(configPath => `    * ${slash(configPath)}`),
```

might fix the windows test, and we usually try to use posix separators all over anyways 🙂

Reply:
the import is missing, fwiw